libraries/mbed/targets/hal/TARGET_Atmel/TARGET_SAM_CortexM0+/TARGET_SAMD21J18A/PeripheralNames.h

```diff
@@ -0,0 +1,158 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
```

Comment:
@0xc0170  yeah. sure. Also we have corrections for CMSIS files, to make it in KnR coding standard. Shall make these changes along correction of years in the next PR.